ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-openshift-logging-6.1.yaml

```diff
@@ -46,7 +46,7 @@ base_images:
 releases:
   latest:
     release:
-      architecture: amd64
+      architecture: multi
```

Comment:
I would suggest to update the file name to `openshift-openshift-tests-private-release-4.16__multi-stable-openshift-logging-6.1.yaml`

Reply:
Or arm64?

Reply:
Ok